### PR TITLE
Workflow blog post

### PR DESCRIPTION
**Pull request by lebauce**

### CODE REVIEW

**Review by nplanel — APPROVED** [review]

File: blog/index.md [hunk 1]
```diff
@@ -3,11 +3,22 @@ title: Blog
 layout: blog-index
 ---
 
+# [Introduction to Skydive workflows](/blog/introduction-to-workflows.html)
+## by Sylvain Baubeau, 29/08/2018
+
+Since version 0.19, Skydive allows you to automate Skydive actions using a new type of object called workflows.
+Let's image you want to test the connectivity between 2 containers. If you had to do it manually, you would have to :
+- create a capture on the interface of each container
+- generate some traffic using the package injector
```
Comment (masco) [inline blog/index.md]: s/package/packet

File: blog/introduction-to-workflows.md [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+title: Introduction to Skydive workflow
+layout: blog-post
+author: Sylvain Baubeau
+date: 29/08/2018
+---
+
+Since version 0.19, Skydive allows you to automate Skydive actions using a new type of object called workflows.
+Let's image you want to test the connectivity between 2 containers. If you had to do it manually, you would have to :
+- create a capture on the interface of each container
+- generate some traffic using the package injector
```
Comment (masco) [inline blog/introduction-to-workflows.md]: same

File: blog/index.md [hunk 3]
```diff
@@ -3,11 +3,22 @@ title: Blog
 layout: blog-index
 ---
 
+# [Introduction to Skydive workflows](/blog/introduction-to-workflows.html)
+## by Sylvain Baubeau, 29/08/2018
+
+Since version 0.19, Skydive allows you to automate Skydive actions using a new type of object called workflows.
+Let's image you want to test the connectivity between 2 containers. If you had to do it manually, you would have to :
```
Comment (masco) [inline blog/index.md]: s/image/imagine

File: blog/introduction-to-workflows.md [hunk 4]
```diff
@@ -0,0 +1,196 @@
+---
+title: Introduction to Skydive workflow
+layout: blog-post
+author: Sylvain Baubeau
+date: 29/08/2018
+---
+
+Since version 0.19, Skydive allows you to automate Skydive actions using a new type of object called workflows.
+Let's image you want to test the connectivity between 2 containers. If you had to do it manually, you would have to :
```
Comment (masco) [inline blog/introduction-to-workflows.md]: same

File: blog/introduction-to-workflows.md [hunk 5]
```diff
@@ -0,0 +1,196 @@
...
+- the other one will be called if an error occurred
+
+Promises are particularly usefull when you need to chain calls and that's exactly what we are going to do with
+the real workflow we want to implement in this blog post.
+
+But before that, we will create a simple workflow based on this simple JavaScript example:
+
+## Our first workflow
+
+A Skydive workflow is composed of:
```
Comment (masco) [inline blog/introduction-to-workflows.md:109]: how about mentioning if any of the below parms are optional?

File: blog/index.md (hunk 1 above)
Comment (lebauce) [reply blog/index.md]: Fixed

**Pull request merged**